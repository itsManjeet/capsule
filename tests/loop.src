let i = 5;
for i > 0 {
    println (i);
    i = i - 1;
}

i = 5;
for i > 0 {
    i = i - 1;

    if i == 2 {
        continue;
    }

    println (i);
}

i = 5;
for i > 0 {
    let j = 5;
    for j > 0 {
        println (i, j);
        j = j - 1;
    }
    i = i - 1;
}

i = 5;
for i > 0 {
    i = i - 1;
    if i == 3 {
        continue;
    }
    j = 5;
    for j > 0 {
        j = j - 1;
        if j == 2 {
            continue;
        }
        println (i, j);
    }
}