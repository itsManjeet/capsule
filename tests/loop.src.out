5
4
3
2
1
4
3
1
0
55
54
53
52
51
45
44
43
42
41
35
34
33
32
31
25
24
23
22
21
15
14
13
12
11
44
43
41
40
24
23
21
20
14
13
11
10
04
03
01
00
